lib/sift/value_parser.rb

```diff
@@ -53,7 +53,7 @@ def parse_as_boolean?
     end
 
     def boolean_value
-      if Rails.version.starts_with?("5")
+      if Rails.version.starts_with?('5') || Rails.version.starts_with?('6')
```

Comment:
Would this work to automatically support future versions beyond 6?
```suggestion
      if Rails.version.to_i >= 5
```

Reply:
It would! Would like to do something more robust to handle different versions in the future, but I like this a lot better!